packages/salesforcedx-vscode-core/package.json

```diff
@@ -29,7 +29,7 @@
     "@salesforce/salesforcedx-utils-vscode": "48.21.0",
     "@salesforce/source-deploy-retrieve": "1.0.10",
     "adm-zip": "0.4.13",
-    "applicationinsights": "1.0.6",
```

Comment:
There's an [issue](https://github.com/microsoft/ApplicationInsights-node.js/issues/433) with applicationinsights version 1.0.6 and a newer `@types/node` version.